cognite/client/data_classes/sequences.py

```diff
@@ -145,3 +179,136 @@ def remove(self, value: List) -> SequenceUpdate:
+        """Get a column by externalId
+        """
+        try:
+            ix = self.column_external_ids().index(item)
+        except ValueError as e:
+            raise ValueError(
+                "Column {} not found, Sequence column external ids are {}".format(item, self.column_external_ids())
+            )
+        return [r[ix] for r in self.values]
+
+    def __iter__(self) -> Generator[Tuple[int, List[Union[int, str, float]]], None, None]:
+        for row, values in zip(self.row_numbers, self.values):
+            yield row, values
+
+    def dump(self, camel_case: bool = False) -> Dict[str, Any]:
+        """Dump the sequence data into a json serializable Python data type.
+
+        Args:
+            camel_case (bool): Use camelCase for attribute names. Defaults to False.
+
+        Returns:
+            List[Dict[str, Any]]: A list of dicts representing the instance.
+        """
+        dumped = {
+            "id": self.id,
+            "external_id": self.external_id,
+            "columns": self.columns,
+            "rows": [{"rowNumber": r, "values": v} for r, v in zip(self.row_numbers, self.values)],
+        }
+        if camel_case:
+            dumped = {utils._auxiliary.to_camel_case(key): value for key, value in dumped.items()}
+        return {key: value for key, value in dumped.items() if value is not None}
+
+    def to_pandas(self, column_names="externalIdIfExists") -> "pandas.DataFrame":
+        """Convert the sequence data into a pandas DataFrame.
+
+        Args:
+            column_names (str): Which field to use as column header. Either "externalId", "id" or "externalIdIfExists" for externalId if it exists and id otherwise.
+
+        Returns:
+            pandas.DataFrame: The dataframe.
+        """
+        pd = utils._auxiliary.local_import("pandas")
+        if column_names == "externalId":
+            identifiers = self.column_external_ids()
+        elif column_names == "id":
+            identifiers = self.column_ids()
+        elif column_names == "externalIdIfExists":
+            identifiers = [eid or id for eid, id in zip(self.column_external_ids(), self.column_ids())]
+        else:
+            raise ValueError("column_names must be 'externalId' or 'id'")
+        return pd.DataFrame(
+            [[x or math.nan for x in r] for r in self.values], index=self.row_numbers, columns=identifiers
+        )
+
+    def column_ids(self):
+        """Retrieves list of column ids for the sequence, for use in e.g. data retrieve or insert methods
+
+        Returns:
+            List of sequence column ids.
+        """
+        return [c.get("id") for c in self.columns]
+
+    def column_external_ids(self):
+        """Retrieves list of column external ids for the sequence, for use in e.g. data retrieve or insert methods
+
+        Returns:
+            List of sequence column external ids
+        """
+        return [c.get("externalId") for c in self.columns]
+
+    def column_value_types(self):
```

Comment:
Missing return type hints on these helper methods. Also I think these should be properties.

Reply:
Still missing these.